### PR TITLE
fix(displays): backlight bleed lighting cleanup and removed home cockpit mode

### LINKED ISSUE #7854: Popout ISIS see-through with no ELEC power
### Aircraft Version

Dev

### Build info

```json
{
    "built": "2023-03-04T16:44:07+00:00",
    "ref": "refs/heads/master",
    "sha": "4a38c4328bcbeb62b8d91ae5ce7f2d6560a0ab11",
    "actor": "tracernz",
    "event_name": "manual",
    "pretty_release_name": "master:4a38c432",
    "version": "v0.10.0-dev.4a38c43"
}
```

### Describe the bug

If you pop out the ISIS I can see a image outside the aircraft (if I look arround the image is also changing.
I'm using homecockpit mode in the EFB and all other popouts are black.

ISIS in the virtual cockpit is black

### Expected behavior

Black background of ISIS in popout

### Steps to reproduce

Use FBW A32NX and pop out ISIS

### References (optional)

image

### Additional info (optional)

_No response_

### Discord Username (optional)

_No response_

### LINKED ISSUE #7936: Display self test for msfs avionics instrument broken
### Aircraft Version

Dev/Exp

### Build info

```json
{
    "built": "2023-02-02T09:20:09+00:00",
    "ref": "HEAD",
    "sha": "fa3b5e6d545c188c3ff6c596655c2f1acaba33bb",
    "actor": "Saschl",
    "event_name": "manual",
    "pretty_release_name": "experimental:fa3b5e6d",
    "version": "v0.10.0-exp.fa3b5e6"
}
```

### Describe the bug

Seems like https://github.com/flybywiresim/a32nx/commit/0128220b4a299aa6a172a65d8091079334f7f119#diff-82000ba2ab1fe4d3630f1b900e314818002445fa45d58e9642da9adcd3a6d1e1R67-R80 introduced a bug that prevents msfs avionics from displaying the self test screen.

### Expected behavior

Self test screen showing after plane/display is being powered up.

Currently, the added logic for the DMC maintenance screens is running in a loop, which interferes with the event driven approach we use to determine the other states of the screen (e.g. self-test).

The following might act as a rough guide to fix the issue:

1. Move the determination of the ```L:A32NX_DMC_DISPLAYTEST:${this.supplyingDmc}``` to a SwitchableProvider, probably a new one is needed (example is AdirsValueProvider from the PFD)
2. Create a sub for the variable, provide as a variable to displayUnit (or even subscribe there directly). Then set the state like it is done with power and brightness

### Steps to reproduce

1. Launch C&D
3. Turn on ext power
4. PFD and EWD immediately are "ready", or the self test screen disappears after roughly 1 second instead of the configured time

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

_No response_

### PR DESCRIPTION
**Pull request by 2hwk**
Fixes #7854
Fixes #7936 

## Summary of Changes

- Fix ISIS background being clear when A/C is powered off
- Fix some background elements still using #040405/ RGB(4,4,5) instead of 040404/RGB(4,4,4)
- Removed home cockpit mode as it is now has no effect (backlight bleed is no longer done through css or in code at all)
- Hotfix for self-test being non-functional (not a proper fix yet)
- New glass texture + comp (sans dust) from Big Mike

## Screenshots (if necessary)

image
image
image
image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

**NOTE: Self-Test is currently broken in Dev, this PR does not fix it**

- Load in several spawn states (C/D, CRZ, Final approach, etc.) and ensure that all the MFDs (PFD/ND), the flyPad EFB and ECAM display elements is working correctly.
- When the A/C is powered off, pop-out all displays with RALT+Click and ensure that the backgrounds are all BLACK and not transparent/reflective.
- Check that self-test is working (include 10s standby time from when displays are turned off. If more than 10 seconds, this will re-trigger self-test)
- Check that interior lighting (pedestal, floods, etc) still works as before.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by 2hwk** [comment]
note: tried to move the ISIS fix to the root element but it just broke more things because of the way auto-brightness/brightness has been implemented. Too much technical debt to clear here, this way works.

File: fbw-a32nx/src/systems/instruments/src/MsfsAvionicsCommon/displayUnit.tsx [hunk 1]
```diff
@@ -112,11 +89,31 @@ export class DisplayUnit extends DisplayComponent<DisplayUnitProps> {
...
+    }
+    */
+
     updateState() {
         if (this.state !== DisplayUnitState.Off && this.failed) {
             this.state = DisplayUnitState.Off;
             clearTimeout(this.timeOut);
-        } else if (this.state === DisplayUnitState.On && (this.brightness === 0 || !this.powered)) {
+        } else if (
+            this.state === DisplayUnitState.On && (this.brightness === 0 || !this.powered)) {
```
Comment (tracernz) [inline fbw-a32nx/src/systems/instruments/src/MsfsAvionicsCommon/displayUnit.tsx:116]:
```suggestion
        } else if (
            this.state === DisplayUnitState.On && (this.brightness === 0 || !this.powered)
        ) {
```

**Review by tracernz — APPROVED** [review]

File: fbw-a32nx/src/systems/instruments/src/MsfsAvionicsCommon/displayUnit.tsx [hunk 2]
```diff
@@ -63,20 +59,6 @@ export class DisplayUnit extends DisplayComponent<DisplayUnitProps> {
...
-            case 1:
-                this.state = DisplayUnitState.MaintenanceMode;
-                break;
-            case 2:
-                this.state = DisplayUnitState.EngineeringTest;
-                break;
-            default:
-                this.state = DisplayUnitState.On;
-            }
             this.updateState();
```
Comment (Saschl) [inline fbw-a32nx/src/systems/instruments/src/MsfsAvionicsCommon/displayUnit.tsx:62]: We should remove this call to updateState too.

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/A320_Neo_EICAS.html [hunk 3]
```diff
@@ -5,17 +5,6 @@
     <glasscockpit-highlight id="highlight"></glasscockpit-highlight>
```
Comment (Saschl) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/A320_Neo_EICAS.html:5]: Let's remove this file completely :D It's not use anymore

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/EICAS_Common.js [hunk 4]
```diff
@@ -35,8 +35,6 @@ class EICASCommonDisplay extends Airliners.EICASTemplateElement {
         this.refreshISA(Arinc429Word.empty());
```
Comment (Saschl) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/EICAS_Common.js:35]: Can be removed

**Pull request merged**